src/chainparams.h

```diff
@@ -102,6 +105,7 @@ class CChainParams
     bool fMineBlocksOnDemand;
     bool fTestnetToBeDeprecatedFieldRPC;
     Checkpoints::CCheckpointData checkpointData;
+    std::vector<std::string> vFoundersRewardAddress; // vector of 2-of-3 multisig P2SH addresses
```

Comment:
The comment here should probably be moved to the array definitions in chainparams.cpp, since I don't think the code depends on the address format (although the script might).